packages/elements/src/components/FileInput/index.tsx

```diff
@@ -23,6 +23,29 @@ export interface FileInputProps {
   inputProps?: Record<string, any>
   required?: boolean
   onFilenameClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void
+  // to integrate with other components
+  afterLoadedImage?: (base64: string) => any
+  croppedImage?: string
+}
+
+export const handleChangeCroppedImage = ({
+  field,
+  croppedImage,
+  setFileName,
+  inputFile,
+}: {
+  field: FieldInputProps<string>
+  croppedImage?: string
+  setFileName: React.Dispatch<string>
+  inputFile: React.RefObject<HTMLInputElement>
+}) => () => {
+  field.onChange({ target: { value: croppedImage ?? '', name: field.name } })
+  if (croppedImage === '') {
```

Comment:
could we change to avoid if inside if
```
  if (inputFile.current && croppedImage === '') {
      inputFile.current.value = ''
    }
  if (croppedImage === '') {
    setFileName('')
  }
```